test/testbindings.py

```diff
@@ -88,6 +88,21 @@ def testAtomMapsAfterDeletion(self):
         mol.removeh()
         self.assertEqual(mol.write("smi", opt={"a":True}).rstrip(), "C[NH:2]")
 
+    def testImplicitCisDblBond(self):
```

Comment:
This came up in the coordinate generation discussions... what should the code do when presented with a SMILES with a trans indication in a small ring? Right now, we don't have any sort of stereo warning.

Reply:
Actually I almost did this. Well I did it, and then reverted, to be honest. But I can put it back. My only concern is that while this is efficient for a single atom, I think a different approach would be better if you are interested in identifying all atoms in smallish rings (I had a discussion on this with @johnmay).

But for sure, there's no harm in making the method available.

Reply:
Sure. It could be part of the PerceiveStereo.